### PR TITLE
SVOM Mission Page

### PR DESCRIPTION
**Pull request by Vidushi-GitHub**
# Description
PR for adding SVOM mission page over GCN website.
 
SVOM team feedback and .svg logo is requested.

# Related Issue(s)

Closing #2175 , as had 200+ merge conflicts.
https://github.com/Vidushi-GitHub/gcn.nasa.gov/pull/3 is also implemented @https://github.com/timroland

### CODE REVIEW

**Comment by Vidushi-GitHub** [comment]
Attached Mission Page 
Screenshot 2024-07-30 at 2 26 21 PM

**Review by lpsinger — COMMENTED** [review]
Why is this a work in progress?

File: app/routes/missions.svom/route.mdx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+ <div className="overflow-table">
+
+| Instruments                        | Energy Range            | Field of View | Localization                                   |
+| ---------------------------------- | ----------------------- | ------------- | ---------------------------------------------- |
+| ECLAIRs                            | 4 - 250 keV             | 2 ster        | ~8' radius (statistical, 90% confidence)       |
+| Gamma Ray burst Monitor (GRM)      | 15 - 5000 keV           | 2.6 ster      | ~1-10° radius (statistical, 90% confidence)    |
+| Microchannel X-ray Telescope (MXT) | 15 - 5000 keV           | 1.1° x 1.1°   | ~10-100'' radius (statistical, 90% confidence) |
+| Visual Telescope (VT)              | Blue Band: 450 - 650 nm | 26' x 26'     | ~TBD'' radius (statistical, 90% confidence)    |
+|                                    | Red Band: 650 - 1000 nm |               |                                                |
```
Comment (jracusin) [inline app/routes/missions.svom/route.mdx:40]: This needs to also have the VT details, or be on the same row in the table as the blue info.

File: app/routes/missions.svom/route.mdx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+ </div>
+
+#### Yearly Trigger Rates
+
+ <div className="overflow-table">
+
+| Instrument        | Type                             | Yearly Rates |
+| ----------------- | -------------------------------- | ------------ |
+| ECLAIRs           | GRBs (TBD)                       | TBD          |
+| Gamma Ray Monitor | GRBs, and other transients (TBD) | TBD          |
```
Comment (jracusin) [inline app/routes/missions.svom/route.mdx:65]: (TBD) not needed if it says "and other transients)

Use either full names or acronyms, not a combination, so "Gamma ray Monitor" should be "GRM"

File: app/routes/missions.svom/route.mdx (hunk 1 above)
Comment (Vidushi-GitHub) [inline app/routes/missions.svom/route.mdx:40]:
```suggestion
| Visual Telescope (VT)                               | Red Band: 650 - 1000 nm |  26' x 26'              |     ~TBD'' radius (statistical, 90% confidence)                                       |
```

File: app/routes/missions.svom/route.mdx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+ <div className="overflow-table">
+
+| Instruments                        | Energy Range            | Field of View | Localization                                   |
+| ---------------------------------- | ----------------------- | ------------- | ---------------------------------------------- |
+| ECLAIRs                            | 4 - 250 keV             | 2 ster        | ~8' radius (statistical, 90% confidence)       |
+| Gamma Ray burst Monitor (GRM)      | 15 - 5000 keV           | 2.6 ster      | ~1-10° radius (statistical, 90% confidence)    |
+| Microchannel X-ray Telescope (MXT) | 15 - 5000 keV           | 1.1° x 1.1°   | ~10-100'' radius (statistical, 90% confidence) |
+| Visual Telescope (VT)              | Blue Band: 450 - 650 nm | 26' x 26'     | ~TBD'' radius (statistical, 90% confidence)    |
+| Visual Telescope (VT)                               | Red Band: 650 - 1000 nm |  26' x 26'              |     ~TBD'' radius (statistical, 90% confidence)                                       |
```
Comment (Vidushi-GitHub) [inline app/routes/missions.svom/route.mdx:40]:
```suggestion
| Visual Telescope (VT)                | Red Band: 650 - 1000 nm |  26' x 26'              |  ~TBD'' radius (statistical, 90% confidence)                                       |
```

File: app/routes/missions.svom/route.mdx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+The SVOM spacecraft carries 4 main instruments of which 2 are French (ECLAIRs and MXT) and 2 are Chinese (GRM and VT). Transient alert messages will be sent for:
+
+ <div className="overflow-table">
+
+| Instruments                        | Energy Range            | Field of View | Localization                                   |
+| ---------------------------------- | ----------------------- | ------------- | ---------------------------------------------- |
+| ECLAIRs                            | 4 - 250 keV             | 2 ster        | ~8' radius (statistical, 90% confidence)       |
+| Gamma Ray burst Monitor (GRM)      | 15 - 5000 keV           | 2.6 ster      | ~1-10° radius (statistical, 90% confidence)    |
+| Microchannel X-ray Telescope (MXT) | 15 - 5000 keV           | 1.1° x 1.1°   | ~10-100'' radius (statistical, 90% confidence) |
```
Comment (Vidushi-GitHub) [inline app/routes/missions.svom/route.mdx:38]: @timroland please provide the feedback for correct energy range of MXT.

**Comment by Vidushi-GitHub** [comment]
Screenshot 2024-10-13 at 23 25 04
Screenshot 2024-10-13 at 23 25 13

@timroland could you please check the content/

**Review by lpsinger — CHANGES_REQUESTED** [review]
This PR touches lots of unrelated files. This looks like an incomplete rebase attempt.

**Comment by Vidushi-GitHub** [comment]
> This PR touches lots of unrelated files. This looks like an incomplete rebase attempt.

Yes, I din't squash properly. Shall I close this and make a fresh PR or rebase again?

**Comment by lpsinger** [comment]
> > This PR touches lots of unrelated files. This looks like an incomplete rebase attempt.
> 
> Yes, I din't squash properly. Shall I close this and make a fresh PR or rebase again?

Whatever you know how to do and are comfortable with.

**Comment by Vidushi-GitHub** [comment]
#2637 PR is opened.

**Pull request closed without merging**